includes/ucf-events-common.php

```diff
@@ -193,10 +193,14 @@ function ucf_events_display_modern( $items, $title ) {
 						<?php echo $event->title; ?>
 					</a>
 				</div>
-				<div class="ucf-event-location-wrapper">
-					<a class="ucf-event-location" href="<?php echo $event->location_url; ?>">
-						<?php echo $event->location; ?>
-					</a>
+				<div class="ucf-event-description-wrapper">
+					<?php
+						if( mb_strlen( $event_description = $event->description ) > $count = 250 ) {
```

Comment:
Is there any particular difference in using mb_strlen() here instead of just strlen()?